test/token/ERC20/utils/ERC1363Holder.test.js

```diff
@@ -0,0 +1,48 @@
+const { BN } = require('@openzeppelin/test-helpers');
+const { expect } = require('chai');
+
+const ERC1363Holder = artifacts.require('$ERC1363Holder');
+const ERC1363 = artifacts.require('$ERC1363');
+
+contract('ERC1363Holder', function (accounts) {
+  const [owner, spender] = accounts;
+
+  const name = 'My Token';
+  const symbol = 'MTKN';
+  const balance = new BN(100);
+
+  beforeEach(async function () {
+    this.token = await ERC1363.new(name, symbol);
+    this.receiver = await ERC1363Holder.new();
+
+    await this.token.$_mint(owner, balance);
+  });
+
+  describe('receives ERC1363 token transfers', function () {
+    it('via transferAndCall', async function () {
+      await this.token.methods['transferAndCall(address,uint256)'](this.receiver.address, balance, { from: owner });
```

Comment:
for completeness, we should check that the event is emitted

Reply:
Added test for events